raiden/settings.py

```diff
@@ -95,6 +95,7 @@
 MIN_MONITORING_AMOUNT_DAI = TokenAmount(1 * 10 ** 18)  # naive approach about 1$
 MIN_MONITORING_AMOUNT_WETH = TokenAmount(4 * 10 ** 15)  # naive approach about 1$
 
+MIN_REVEAL_TIMEOUT = BlockTimeout(20)
```

Comment:
Here I'm unsure how this should be implemented.
Right now this is a global constant, that is directly imported from `raiden.settings`.
This is problematic for different environments, where different minimum reveal timeouts could be necessary.
The tests already have this problem, since we go for low reveal timeouts because of PoA.

Now, should we:
- monkey patch the constant in the tests (doesn't sound good)
- add as a config parameter to RaidenConfig
- unify `DEFAULT_REVEAL_TIMEOUT` and `MIN_REVEAL_TIMEOUT`, with the reasoning that the default should be the minimum, and the minimum should be "safe"?

Reply:
Hm, I wonder if that should be something that is defined in the smart contract...
Let's talk about it in the standup.

Reply:
So now this is global mapping, where the value is dependent on the chain_id.
During accessing the mapping, when there is no min reveal timeout for the given chain, it will be simply set to 1 (which is how it was before).
The values are all set to 20 now, but someone with more insight might change that value later on...